spacy/pipeline/span_finder.py

```diff
@@ -1,8 +1,7 @@
 from functools import partial
 from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple, cast
 
-from thinc.api import (Config, Model, Ops, Optimizer, get_current_ops,
-                       set_dropout_rate)
+from thinc.api import Config, Model, Ops, Optimizer, get_current_ops, set_dropout_rate
```

Comment:
I ran `black` first then `isort` and then the PR got rejected so I had to run `black` again. Do we have to run `isort` first than `black`?